<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UITextFieldPropertyGridWidget</class>
 <widget class="QWidget" name="UITextFieldPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>129</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="textLineEdit">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>26</y>
      <width>245</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="fontSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>60</y>
      <width>57</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fontNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font:</string>
    </property>
   </widget>
   <widget class="QColorButton" name="textColorPushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>101</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Color:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fontSelectButton">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>52</y>
      <width>197</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:right; font-size: 11px;</string>
    </property>
    <property name="text">
     <string>default font</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorButton</class>
   <extends>QPushButton</extends>
   <header>qcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textLineEdit</tabstop>
  <tabstop>fontSizeSpinBox</tabstop>
  <tabstop>textColorPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
